src/tutorial/TutorialEventArgs.cs

```diff
@@ -62,12 +62,16 @@ public void ReuseEvent(bool usesScreenRelativeMovement, Vector3 movementDirectio
 
 public class EntityPositionEventArgs : TutorialEventArgs
 {
-    public EntityPositionEventArgs(Vector3? position)
+    public EntityPositionEventArgs(Vector3? position, CompoundBag compounds, MicrobeHUD hud)
```

Comment:
Which compounds are these, the entity's or the player's? I think that needs naming clarification. Also I'd totally remove the HUD as I think it just doesn't fit at all into the idea of event args. At most I think a hack with the sender of the event could be done but even that seems not very clean design.

Reply:
I'll rename the compounds. As for the HUD I have no attachment to this pattern, I just didn't see a cleaner way to send it.

Reply:
After reading the code it seems like it is the current player's data, which kind of doesn't fit logically as the position is for another entity and the player data is just tacked on... So I would really prefer if that data was also stripped so that we follow more separation of concerns (disallowing the tutorial from directly messing with the player state).